Standardize hotkey help text

Niloth-p opened the pull request:
### What does this PR do, and why?
Improves the conformity of help descriptions.

### Outstanding aspect(s)
[ ] Usage of 'the' needs to be made uniform too. This has not yet been addressed.

### External discussion & connections
- [x] Discussed in **#zulip-terminal** inside `re-categorize` [here](https://chat.zulip.org/#narrow/stream/206-zulip-terminal/topic/Re-Categorization.20of.20Hotkeys/near/1793558) and [here](https://chat.zulip.org/#narrow/stream/206-zulip-terminal/topic/Re-Categorization.20of.20Hotkeys/near/1797363)
- [ ] Fully fixes #
- [ ] Partially fixes issue #
- [ ] Builds upon previous unmerged work in PR #
- [ ] Is a follow-up to work in PR #
- [ ] Requires merge of PR #
- [ ] Merge will enable work on #

### How did you test this?
- [ ] Manually - Behavioral changes
- [x] Manually - Visual changes
- [ ] Adapting existing automated tests
- [ ] Adding automated tests for new behavior (or missing tests)
- [ ] Existing automated tests should already cover this (*only a refactor of tested code*)

### Self-review checklist for each commit
- [x] It is a [minimal coherent idea](https://github.com/zulip/zulip-terminal#structuring-commits---speeding-up-reviews-merging--development)
- [x] It has a commit summary following the [documented style](https://github.com/zulip/zulip-terminal#structuring-commits---speeding-up-reviews-merging--development) (title & body)
- [x] It has a commit summary describing the  motivation and reasoning for the change
- [x] It individually passes linting and tests
- [ ] It contains test additions for any new behavior
- [ ] It flows clearly from a previous branch commit, and/or prepares for the next commit

zormit commented on zulipterminal/config/keys.py:
```diff
@@ -154,7 +154,7 @@ class KeyBinding(TypedDict):
     'ADD_REACTION': {
         'keys': [':'],
-        'help_text': 'Show/hide Emoji picker popup for current message',
+        'help_text': 'Show/hide Emoji Picker popup for current message',
```
I'm not sure on this one, given that "Emoji Picker" is not the title used for the popup (which is different to the other cases above).

My personal preference would be to use "Emoji-picker" instead of "Emoji picker popup" and "Emoji-picker popup", but any of these three could work.

Niloth-p replied:
Yes, that's a tricky one indeed.
Yep, we could remove the 'popup' word. Good one.
Googling shows a usage preference of not using a hyphen. So, I went with that.
And yes, the popup does not have a header.

zormit reviewed (approved):
Generally looks good to me and makes things indeed more consistent. Thanks for working on this. I left one comment where my personal opinion is a bit different, but it's on the level of [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality) so I certainly won't insist on it.

My approval means that I don't need to look again at this after this one thing is resolved, but of course I'll leave the final decision to @neiljp.

Niloth-p commented:
Just updated the "Emoji Picker Popup" mentions.
commit 1 (case consistency): Lower cased the emoji picker.
commit 2 (update wording): Removed popup.
Updated both commit messages.

rsashank commented:
LGTM - seems pretty clear :)

neiljp commented:
@Niloth-p Thanks for this again - looking forward to more iterations along these lines and the contextual work! :tada:
